notebooks/Sigma_test_Notebook.ipynb

```diff
@@ -0,0 +1,340 @@
+{
+ "metadata": {
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.8.5-final"
+  },
+  "orig_nbformat": 2,
+  "kernelspec": {
+   "name": "python3",
+   "display_name": "Python 3.8.5 64-bit ('.venv')",
+   "metadata": {
+    "interpreter": {
+     "hash": "74f679a4fc854d75b381693882d369b7543a476b4b9cfda76eebb965e9131c3c"
+    }
+   }
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2,
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#!pip install -q timesketch_api_client"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pip install ../api_client/python"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from timesketch_api_client import search\n",
+    "from picatrix.lib import state as state_lib\n",
+    "\n",
+    "import altair as alt\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "from timesketch_api_client import config\n",
+    "from timesketch_import_client import helper\n",
+    "from timesketch_import_client import importer\n",
+    "from timesketch_api_client import search\n",
+    "\n",
+    "\n"
+   ]
+  },
+  {
+   "source": [
+    "# Create the sketch and import some data"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ts_client = config.get_client(confirm_choices=False)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ts_client.create_sketch(name=\"test\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sketch = ts_client.get_sketch(1)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import_helper = helper.ImportHelper() \n",
+    "\n",
+    "with importer.ImportStreamer() as streamer:\n",
+    "  streamer.set_sketch(sketch)\n",
+    "  streamer.set_config_helper(import_helper) \n",
+    "\n",
+    "  streamer.set_timeline_name('sigma_events.jsonl')\n",
+    "\n",
+    "  streamer.add_file('../test_tools/test_events/sigma_events.jsonl')"
```

Comment:
there is no guarantee here that this file exists... this depends on how you start your notebook, how it is configured, and the paths you've got there....

you can also fetch the file from GH to ensure it's there and correct

Reply:
GH download it is